### DISCUSSION TITLE
Any working example of blob?

### DISCUSSION THREAD

**Discussion by ari1337an** [simolus3/drift#2848] [General]
An example of storing single image as an blob would suffice. I know I can store images as base64 string, but I don't think its the best idea.

**Comment by simolus3** [comment]
You can store images in a blob by converting them to bytes. If you have been converting them to base64 strings before, you pretty must just have to not do that last conversion step. If you have a `List<int>`, you can use `Uint8List.fromList` to convert that into the `Uint8List` expected by drift.

I can come up with an example. How are you obtaining images to store in the database though? Do you get them as a PNG from somewhere or do you have an `Image` from `dart:ui` you want to store?
